cubic-server/default/Overworld.cpp

```diff
@@ -2,6 +2,8 @@
 #include "../world-storage/WorldStorage.hpp"
 #include "../world-storage/ChunkColumn.hpp"
 
+#define NB_SPAWN_CHUNKS 19
```

Comment:
Please use a constexpr variable for stuff like this.
It allows for proper typechecking.